templates/terraform/tests/resource_compute_router_test.go

```diff
@@ -0,0 +1,216 @@
+package google
```

Comment:
In Puppet/Chef/Ansible, we have the products/compute/files folder to hold files exclusive to compute.

I feel like Terraform should move towards that model as well for consistency, unless this test template will be used for other things too.

Reply:
Yeah, we've talked a little bit about this- on one hand, yes. On the other hand, all the other go code that we have for terraform is inside the templates folder (a lot of it, though not actually a template, is shared), and it feels weird to stick this somewhere totally far away in the code base just because of that. This will be a good thing to revisit when we go through and scrutinize the names of things and stuff.